setup.cfg

```diff
@@ -29,6 +29,7 @@ package_dir =
 packages = find_namespace:
 install_requires =
     stactools ~= 0.2.1a1
+    jupyter
```

Comment:
Does it make sense to move a package requirement to requirements-dev.txt for running part of the code that isn't for development? If you are content leaving this requirement in setup.cfg we can change this requirement to jupyterlab==3.0.16 so that the version is pinned.

Reply:
The `docker/notebook` script runs the dev image.
Ideally we'd have a third state for Jupyter. It doesn't make sense for it to be part of the main dependencies since the stactools CLI doesn't need Jupyter, but it's not really a `dev` dependency either. But that's probably more complicated than necessary, and putting it in the dev dependencies is easy.

Reply:
That makes sense. I've moved the requirement as requested with the latest commit.